docs/configuration.md

```diff
@@ -1,15 +1,24 @@
 # Configure the bot
 
-This page explains how to configure your `config.json` file.
+This page explains how to configure your configuration file.
 
-## Setup config.json
+Per default, the bot loads configuration from the `config.json` file located in the current working directory.
+You can change the configuration file used by the bot with the `-c/--config` option.
 
-We recommend to copy and use the `config.json.example` as a template
+If you used the [Quick start](installation.md/#quick-start) method for installing 
+the bot, the installation script should have already created the default configuration file (`config.json`) for you.
+
+We recommend you to copy and use the `config.json.example` as a template
```

Comment:
do we? 

that's true for bittrex - but for binance i would want to use the `config_binance.json.example` file as template ... ?

Reply:
I only changed "recomment to copy" --> "recommend you to copy" in this sentence.

Actually I guess the installation procedure (the part which prepares the default config) should ask for exchange and use diff. template configs, if they are found in the distribution... But this is not for this PR...